chamber-of-commerce.md

```diff
@@ -0,0 +1,93 @@
+# Woodcrest Chamber of Commerce
+## Business Approval Process
+*Effective Date: [Insert Date]*
+
+### 1. Eligibility
+1.1 Any individual who wishes to open a business in Woodcrest must be a member in good standing of the community.
+1.2 Businesses must be operated on designated commercial rental plots within the Woodcrest.
+1.3 Individuals applying for a business license must not have any criminal history recorded with the Woodcrest Police Department.
+1.4 Individuals must not have any outstanding fines or unresolved infractions with the Woodcrest Police Department at the time of application.
+
+### 2. Application Process
+2.1 Individuals must submit a business proposal to the Chamber of Commerce for review and approval before renting a commercial plot.
+2.2 The proposal must include:
+   a) Business name and type (e.g. retail, service, entertainment, etc.)
+   b) Detailed description of products/services offered
+   c) Target customer base and how it fits into the Woodcrest community
+   d) Proposed location (specific rental plot if known)
+   e) Any special requirements or requests (e.g. zoning variances)
+   f) Proof of any necessary permits or licenses (e.g. alcohol permit, health department certifications)
+2.3 Individuals are responsible for obtaining all required permits and licenses from the appropriate Woodcrest and North Carolina authorities before submitting their proposal.
+2.4 Proposals must be submitted via [designated in-game or out-of-game channel].
```

Comment:
I'm going to see if I can have this be something that ties into the website directly, like a form to fill out, which would either send an email or even notify in discord.

Worst case, this can be a mailbox setup like we do for Rush cards, but then it would be a one-way communication via notecard.